powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthOnboardingTest.java

```diff
@@ -274,15 +276,18 @@ void testMaxAttemptsReached() throws Exception {
         // Obtain activation OTP from testing endpoint
         String otpCode = getOtpCode(processId);
 
-        for (int i = 0; i < 5; i++) {
-            assertThrows(AssertionFailedError.class, () ->
-                    createCustomActivation(processId, "0000000000", clientId),
-                    "Activation with invalid OTP should fail");
+        for (int i = 0; i < 4; i++) {
```

Comment:
What about to invert the `for` statement? Just thinking. Isn't it weird?
```suggestion
        for (int i = 4; i > 0; i--) {
```

Reply:
Sure.